native/src/main/java/org/ballerinax/asb/admin/AdminClient.java

```diff
@@ -0,0 +1,949 @@
+import com.azure.messaging.servicebus.administration.models.AuthorizationRule;
+import com.azure.messaging.servicebus.administration.models.EmptyRuleAction;
+import com.azure.messaging.servicebus.administration.models.QueueProperties;
+import com.azure.messaging.servicebus.administration.models.RuleAction;
+import com.azure.messaging.servicebus.administration.models.RuleFilter;
+import com.azure.messaging.servicebus.administration.models.RuleProperties;
+import com.azure.messaging.servicebus.administration.models.SqlRuleAction;
+import com.azure.messaging.servicebus.administration.models.SqlRuleFilter;
+import com.azure.messaging.servicebus.administration.models.SubscriptionProperties;
+import com.azure.messaging.servicebus.administration.models.TopicProperties;
+import io.ballerina.runtime.api.PredefinedTypes;
+import io.ballerina.runtime.api.creators.TypeCreator;
+import io.ballerina.runtime.api.creators.ValueCreator;
+import io.ballerina.runtime.api.types.ArrayType;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.utils.TypeUtils;
+import io.ballerina.runtime.api.values.BArray;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BHandle;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import org.ballerinax.asb.util.ASBConstants;
+import org.ballerinax.asb.util.ASBErrorCreator;
+import org.ballerinax.asb.util.ASBUtils;
+import org.ballerinax.asb.util.ModuleUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static io.ballerina.runtime.api.creators.ValueCreator.createRecordValue;
+
+
+/**
+ * This facilitates the client operations of ASB Administrator client in Ballerina.
+ */
+public class AdminClient {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AdminClient.class);
+    /**
+     * Parameterized constructor for ASB Administrator (ServiceBusAdminClient).
+     *
+     * @param connectionString Azure service bus connection string
+     * @return clientEp Azure Service Bus Administrator instance.
+     */
+    public static Object initializeAdmin(String connectionString) {
+        try {
+            ServiceBusAdministrationClientBuilder adminClientBuilder = new ServiceBusAdministrationClientBuilder()
+                    .connectionString(connectionString);
+            LOGGER.debug("ServiceBusAdminClient initialized");
+            return adminClientBuilder.buildClient();
+        } catch (BError e) {
+            return ASBErrorCreator.fromBError(e);
+        } catch (ServiceBusException e) {
+            return ASBErrorCreator.fromASBException(e);
+        } catch (Exception e) {
+            return ASBErrorCreator.fromUnhandledException(e);
+        }
+    }
+    /**
```

Comment:
Missing blank lines between method definitions

Reply:
Addressed in [f0bbe24](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/f0bbe24dab6c69c66ba7ae68ca343a18c0eaf18b).